### ISSUE TITLE
False positive in SimpleAccessorNameNotation check

### ISSUE THREAD

**Issue by mkordas** [sevntu-checkstyle/sevntu.checkstyle#369]
SimpleAccessorNameNotation check reports the following violation when getter is provided for private static final constant:
```java
class FalsePositive {
    private static final String MY_CONSTANT = "";

    String getMyConstant() { // violation here, wrong getter name
        return MY_CONSTANT;
    }
}
```

**Comment by valfirst** [comment]
+1. I faced with the same issue

**Comment by romani** [comment]
duplicate of #166
